jaseci_ai_kit/install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Let's add some quick instructions  in the readme on how to use this script.

Reply:
On it